docs/content/examples/tutorials/docker-build.md

```diff
@@ -21,19 +21,26 @@ git clone --recurse-submodules https://github.com/docker-mailserver/docker-mails
 
 ### About Docker
 
-#### Version
+#### Minimum supported version
 
-We make use of build-features that require a recent version of Docker. Depending on your distribution, please have a look at [the official installation documentation for Docker](https://docs.docker.com/engine/install/) to get the latest version. Otherwise, you may encounter issues, for example with the `--link` flag for a [`#!dockerfile COPY`](https://docs.docker.com/engine/reference/builder/#copy) command.
```

Comment:
I removed the `--link` example as I think that has a fallback behaviour for compatibility and the information here with link doesn't provide that context.

- The HereDocs feature arrived earlier [around 2021H2](https://www.docker.com/blog/introduction-to-heredocs-in-dockerfiles/)
- The `COPY --link` feature [around 2022Q2](https://www.docker.com/blog/image-rebase-and-improved-remote-cache-support-in-new-buildkit/)

Docker [v23 was released Feb 2023](https://docs.docker.com/engine/release-notes/23.0/#2300). It enables BuildKit by default. It makes for a good baseline (_for building the image_), instead of just "recent version", while being a little vague on earlier version support for brevity.